packages/cdc-react/src/components/Alert/Alert.tsx

```diff
@@ -21,12 +21,14 @@ export const Alert = ({
   leftAlign,
   heading,
   children,
-}: AlertProps) => {
+  className,
+}: AlertProps & JSX.IntrinsicElements["div"]) => {
   const classList = ["usa-alert", `usa-alert--${type}`];
   if (slim) classList.push("usa-alert--slim");
   if (noIcon) classList.push("usa-alert--no-icon");
   if (fullWidth) classList.push("fluid");
   if (leftAlign) classList.push("left-align");
+  if (className) classList.push(className);
```

Comment:
Rather than adding this to the inner div classList, what about adding it to the outer div classList? 

```
<div className={`cdc-react ${className}`}>
```

Reply:
I'm not sure. I thought the purpose of the `cdc-react` class was only to namespace our components and allow us to nest within in like:
```
.cdc-react {
  .my-class-1 {}
  .my-class-2 {}
}
```
but I have seen mixed usage in our codebase. I can see pros and cons of both strategies. Which do you prefer?